awscli/testutils.py

```diff
@@ -639,11 +606,8 @@ def aws(command, collect_memory=False, env_vars=None,
     """
     if platform.system() == 'Windows':
         command = _escape_quotes(command)
-    if 'AWS_TEST_COMMAND' in os.environ:
-        aws_command = os.environ['AWS_TEST_COMMAND']
```

Comment:
I couldn't find it referenced anywhere, it's entirely possible but searching for obvious sub strings did not bring anything up.